runtimeverification/kontrol
# `--break-on-cheatcodes` option to support breaks for assertion cheatcodes

### What's new

Added assertion cheatcode rules from `KONTROL-ASSERTIONS` for `--break-on-cheatcodes` option of `kontrol prove`.

### How to test

Run `kontrol prove --break-on-cheatcodes` for the following test and you will see that there is a break for `assertEq` on this PR branch that is not present before this PR:
```solidity
    function test_assertEq(address a, address b) public pure {
        vm.assume(a == b);
        assertEq(a, b);
    }
```
